src/index.css

```diff
@@ -18,6 +18,7 @@ textarea,
   margin-right: 10px;
   margin-bottom: 10px;
   user-select: initial;
+  width: calc(100% - 22px); /* subtract margin and border */
```

Comment:
If this caused the problem in mobile/responsive (= single column) view, then moving this into the media query should fix it?

Reply:
That's not the reason. The reason is that I put the buttons in a single row, so they don't fit on the screen. I'll add a media query to fix it.